src/main/java/org/jitsi/jicofo/JitsiMeetConference.java

```diff
@@ -1682,4 +1721,81 @@ private String createSharedDocumentName()
 
         return sharedDocumentName;
     }
+
+    /**
+     * (Re)schedules {@link SinglePersonTimeout}.
+     */
+    private void rescheduleSinglePeerTimeout()
+    {
+        if (this.executor != null)
```

Comment:
In this method the properties are accessed with `this` and in the next method the same properties are accessed without `this`. It's better to keep the same code style.